### DISCUSSION TITLE
Terraform as single IaC provider across stack

### DISCUSSION THREAD

**Discussion by mishraomp** [bcgov/quickstart-openshift#2219] [Ideas]
Currently, we have lot of IaC in play, `HELM` , `OC templates`, `SAM`, `Terraform`, et..
This makes it harder for longer term sustainment as one needs to learn syntax for each.
if we make Terraform as the single IaC solutions which works across `public clouds(AWS, Azure, GCP)` as well as for `OpenShift` then we dont have multiple tools to do `CD`

As we have mostly settled on `GHA` for `CI`, I would like to propose `Terraform` as the `CD` tool 

@DerekRoberts @franTarkenton @abschwenker @conbrad @cberg-aot @jon-funk @marcellmueller @paulushcgcj

**Comment by DerekRoberts** [comment]
@mishraomp I haven't tried Terraform for OpenShift, but would be happy to give it a shot. What about OpenTofu?

**Comment by mishraomp** [comment]
not worth the effort for extra tooling, closing this thread
